src/braket/experimental/autoqasm/instructions/measurements.py

```diff
@@ -22,29 +22,32 @@ def my_program():
         measure(0)
 """
 
+from __future__ import annotations
```

Comment:
Should we plan to do this everywhere after moving the repo over?

Reply:
Yes, I've just been doing it piecemeal but it would be a good idea to make a pass through for consistency, either before or after moving the repo. I opened a [task in the backlog here](https://github.com/orgs/amazon-braket/projects/2/views/1?pane=issue&itemId=57874367).